toolchain/check/cpp/import.cpp

```diff
@@ -2268,6 +2267,13 @@ static auto MapConstant(Context& context, SemIR::LocId loc_id,
         context,
         MakeImportedLocIdAndInst<SemIR::IntValue>(
             context, imported_loc_id, {.type_id = type_id, .int_id = int_id}));
+  } else if (auto* bool_literal = dyn_cast<clang::CXXBoolLiteralExpr>(expr)) {
+    inst_id = AddInstInNoBlock(
+        context, MakeImportedLocIdAndInst(
+                     context, imported_loc_id,
+                     SemIR::BoolLiteral{.type_id = type_id,
```

Comment:
nit: We usually write the instruction type in the instruction-creating functions as an explicit template parameter. Why? Idk, I always have to move it to the `{}` when I have an error and need to see what it actually is. But I think clang-format does a nicer job with things that way, and by being consistent you can grep for `<SemIR::SomeInstName` to find where it's created.

```suggestion
        context, MakeImportedLocIdAndInst<SemIR::BoolLiteral>(
                     context, imported_loc_id,
                     {.type_id = type_id,
```

Will probably need to reformat after

Reply:
Done.